deployment/roles/reverse/templates/apache/page/index_debug.html.j2

```diff
@@ -0,0 +1,35 @@
+#jinja2: lstrip_blocks: True
+          <div class="panel panel-default col-md-12 container">
+            <a class="panel-header no-toggle" href="#">
+              <h2>Vitam UI - Debug</h2>
+            </a>
+            <div class="panel-body">
+              {% for item in groups['hosts_vitamui'] %}
+              <div class="row component-line">
+                <div class="col-md-3">{{ item }}</div>
+                <div class="col-md-2">{{ hostvars[item]['ip_admin'] }}</div>
+                <div class="col-md-2">
+                  {% for group in groups %}
+                  {% if item in groups[group] %}
+                  {% if (group not in ('hosts','all','vitam','prometheus','reverse','elasticsearch','mongo_common')) and (not(group is search('^zone'))) %}
+                  {{ group|regex_replace('hosts_', '')| replace('_','-') }}<br/>
+                  {% endif %}
+                  {% endif %}
+                  {% endfor %}
+                </div>
+                <div class="col-md-2">
+                  {% for group in groups %}
+                  {% if group is search('^zone') %}
+                  {% if item in groups[group] %}
+                  {{ group| replace('_',' ') }}<br/>
+                  {% endif %}
+                  {% endif %}
+                  {% endfor %}
+                </div>
+                <div class="col-md-2">
+                  <a href="https://{{ vitam_reverse_external_dns }}/nodes/{{ item }}/browse" class="btn btn-primary btn-xs" target="_blank">Browser</a>
```

Comment:
Si groups['hosts_browse'] | default([]) | length >0 , sinon ne pas afficher ce lien

Reply:
Si on applique cette condition, dans un cas de figure où certains hôtes auraient le browser d'activé et d'autres non, un hôte n'étant pas dans le groupe `hosts_browse` pourrait disposer du bouton. La condition devrait plutôt être `if item in groups['hosts_browse']`. Cela a été appliqué, c'est en cours de test.